### PR TITLE
Linting with eslint

### PR DESCRIPTION
**Pull request by Simounet**
I love linting so this is a proposal. I don't know your habits but it's [eslint](http://eslint.org) kind of classic rules. Let me know if you're ok with that @Rurik19 .
It is a work in progress anyway. Don't need to be merged right away.

### CODE REVIEW

**Comment by tcitworld** [comment]
- [ ]  Needs a `package.json` file for the eslint dependencies

I'm not fond of the `standard` style, I prefer `airbnb-base` but I won't mind.

**Comment by Rurik19** [comment]
OK, I've install ESLint as extension to vscode. 
I really don't think package.json and full bunch of dependencies in node_modules/  is nessessary for our little project. Better have just eslintrc and eslint installed globally.

**Comment by Simounet** [comment]
It is useful for developers to know what we are using easily and which version.

**Comment by Simounet** [comment]
@tcitworld I didn't knew about `airbnb-base`, it's pretty cool and future proof. I just don't like the trailing comma but we can add this to our custom rules.
Plus, with the existing code base, there is a little bit of work to do but I think it could be a great. So let me know what you two prefer.

**Comment by Rurik19** [comment]
Are there some comparison of styles standard/airbnb/google/yandex to see ?

**Comment by Simounet** [comment]
It is not perfect but I found that: https://sqren.github.io/eslint-compare/ .
You can try the style adding the style and use eslint --fix to see what changed.

File: package.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    "extension"
+  ],
+  "author": "wallabagger",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/wallabag/wallabagger/issues"
+  },
+  "homepage": "https://github.com/wallabag/wallabagger#readme",
+  "devDependencies": {
+    "eslint": "^3.12.1"
```
Comment (tcitworld) [inline package.json]: You'll also need the `eslint-config-%style%` dependency.

**Comment by Rurik19** [comment]
I think standard is OK. airbnb requires also too many modules just to work. Why do I need eslint-plugin-react or eslint-plugin-jsx-a11y for this project?
And, @Simounet I think "linebreak-style" must be "warn" (at least :-) ), as I am working on windows.

**Comment by tcitworld** [comment]
> I think standard is OK. airbnb requires also too many modules just to work. Why do I need eslint-plugin-react or eslint-plugin-jsx-a11y for this project?

That's why `airbnb-base` exists for non-react projects. I'm fine with `standard` though.

**Comment by Rurik19** [comment]
Hi. commented lines removed

**Comment by Simounet** [comment]
I tested it and it looks fine if that's okay with you.

**Comment by Rurik19** [comment]
ok

**Pull request merged**